modules/private-clusters-about-ibm-power-vs.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_cloud_public/installing-ibm-cloud-private.adoc
+
+:_content-type: CONCEPT
+[id="private-clusters-about-ibm-power-virtual-server_{context}"]
+= Private clusters in IBM Power Virtual Server
+
+To create a private cluster on IBM Power Virtual Server, you must provide an existing private VPC and subnets to host the cluster. The installation program must also be able to resolve the DNS records that the cluster requires. The installation program configures the Ingress Operator and API server for only internal traffic.
+
+The cluster still requires access to internet to access the IBM Cloud VPC APIs.
+
+The following items are not required or created when you install a private cluster:
+
+* Public subnets
+* Public network load balancers, which support public ingress
+* A public DNS zone that matches the `baseDomain` for the cluster
+
+The installation program does use the `baseDomain` that you specify to create a private DNS zone and the required records for the cluster. The cluster is configured so that the Operators do not create public records for the cluster and all cluster machines are placed in the private subnets that you specify.
```

Comment:
Please replace this with something like:

You will also need to create an IBM DNS service containing a DNS zone that matches your baseDomain. Unlike standard deployments on Power VS which use IBM CIS for DNS, you must use IBM DNS for your DNS service.